README.md

```diff
@@ -12,15 +12,72 @@ npm start
 
 This will watch the `apps` directory for changes and run everything.
 
-## Environment Layout Explained
+### Environment Layout Explained
 In trying to make this a smooth development environment there were a few decisions made.
 First up, unless you're extending the development environment, the only folder you should be concerned about is `apps` and `dist`.
-The other folders are for making the development environment a pleasure to work with (hopefully).
-The `.server` directory is the "mock" server with storage, this is where the code lives which mocks up the Rocket.Chat server.
 
-## Generating an App ID
+### Generating an App ID
 We require UUID Version 4 for IDs. To generate one for your App we recommend this site: https://www.uuidgenerator.net/version4
 
-## Logging Inside an App
+### Logging Inside an App
 Due to limitations of NodeJS's `vm` package we have had to implement a custom logger class.
 To make usage of this you can use `this.getLogger()` and then do the normal `console` style logging.
+
+## Rocket.Chat App Development
+
+The minimum necessary to create a Rocket.Chat App is having an `app.json` file and a `.ts` class which extends the `App` class and an icon.
+
+### npm run create-app
+
+The development tools provide a command to quickly scaffold a new Rocket.Chat App, simply run `npm run create-app "<App Name>"` and a new folder will be created inside the `apps` folder with an `app.json` file containing the basic information of your app (please remember to update the UUID), a dummy `icon.jpg` image and the initial implementation of the App on `index.ts`.
```

Comment:
What do you think about generating a new uuid for them? https://www.npmjs.com/package/uuid4

Reply:
Even better.